### DISCUSSION TITLE
How to create a new badge from the new docausorus documentation ?

### DISCUSSION THREAD

**Discussion by 12rambau** [badges/shields#9318] [Q&A] [answered]
Congratulation for the revamp of the documentation, it looks great. I wanted to create a static shield as I use to, i.e. see in the interactive panel if I'm not making mistakes and play with the options. 

I went to static badges: https://shields.io/badges

My guess is that I need to fill parameters somewhere to "activate" the "execute" button but I don't find where. Is it a wip feature or am I missing something ?

**Comment by chris48s** [comment, marked as answer]
![GIFrecord_2023-06-26_165847](https://github.com/badges/shields/assets/6025893/9c04d330-eedc-4988-bb07-fd4a4e8ad1b8)

**Comment by 12rambau** [comment]
I was so sure I clicked everywhere to find a TextField.... Maybe some a11y improvement to make them pop a bit more but now that I know I won't bother you anymore so thanks a lot, that works like a charm.

**Reply by chris48s** [reply]
It is useful feedback that some people aren't seeing them.

I've opened https://github.com/badges/shields/pull/9324 with an improvement for this
